### ISSUE TITLE
Error during Squeak-trunk update produces misleading error message

### ISSUE THREAD

**Issue by theseion** [hpi-swa/smalltalkCI#352]
When an error occurs during the update process of the Squeak-trunk image (as is currently the case) SmalltalkCI fails with

> Build was unable to report intermediate build status.
> cat: /home/travis/smalltalkCI-master/_builds/build_status.txt: No such file or directory
> Error with status code 1:
> 229 check_and_consume_build_status_file /home/travis/smalltalkCI-master/helpers.sh
> The command "smalltalkci" exited with 1.
> Done. Your build exited with 1.

This issue is reproducible locally, the cause being premature termination of the image (as instructed by the error handler).

While the error message isn't actually wrong, it suggests a problem with SmalltalkCI itself instead of an error in the update process.

**Comment by fniephaus** [comment]
Thanks, I will have a look soon. Have you reported the root cause on squeak-dev?

**Comment by theseion** [comment]
No, I haven't but I saw something about problems with empty packages / package removal, so I assumed that it's a known issue.

**Comment by theseion** [comment]
I've sent a message to the list.

**Comment by fniephaus** [comment]
Thanks!
